src/main/java/io/codelex/flightplanner/controllers/CustomerApiController.java

```diff
@@ -38,6 +38,13 @@ public ResponseEntity<List<Airport>> searchAirports(@RequestParam String search)
                 .body(this.flightPlannerService.searchAirports(search.trim()));
     }
 
+    @GetMapping("/flights")
+    public ResponseEntity<FlightSearchResponse> getFlights() {
```

Comment:
To ResponseEntity nevajag, var vienkārši FlightSearchResponse, tas pats arī citur - ResponseEntity ņem nost un, ja kaut kur vajadzīgs kļūdas kods, tad tur met ResponseStatus exception.